examples/test_data_gen/test_data_gen_local/config.ini

```diff
@@ -0,0 +1,11 @@
+; should_skip_doc_split = True
```

Comment:
Seems just a few required parameters. Would prefer to use command line instead of config.ini, we could distinct required and optional params.